cli/cmd/admin/start.go

```diff
@@ -99,7 +99,7 @@ func StartCmd(ch *cmdutil.Helper) *cobra.Command {
 
 			// Init logger
 			cfg := zap.NewProductionConfig()
-			cfg.Level.SetLevel(conf.LogLevel)
+			cfg.Level.SetLevel(zapcore.InfoLevel)
```

Comment:
I think this needs to be reverted?

Reply:
My bad.. using the wrong variable, just reverted back, thx for pointing out!